components/automate-chef-io/content/docs/applications-setup.md

```diff
@@ -0,0 +1,218 @@
++++
+title = "Setting up the Applications Tab"
```

Comment:
im kind of not a fan of using `Tab` especially in the left nav and heading
i wonder if this could be just `Applications Setup`

Reply:
or i think this is more config info, right? so maybe `Configuring Applications`

Reply:
I agree that "tab" in the nav is :yuck:. I'll find a better title.

Reply:
Dashboard!